buildlib/dockers/docker-compose.yml

```diff
@@ -12,6 +12,17 @@ services:
         MOFED_OS: rhel7.6
         CUDA_VERSION: 11.2.0
         OS_VERSION: 7
+  centos7-gpu-mofed5-cuda11:
```

Comment:
centos7-cuda11-mofed5

Reply:
we already have this name

Reply:
what is the difference in the container image then? maybe unite?

Reply:
added several addition links
```
RUN cd /usr/lib64 && \
    ln -s libudev.so.1 libudev.so && \
    ln -s libz.so.1 libz.so
```
and valgrind package

Reply:
maybe it can be same container?
I think it's better to not have 2 unrelated containers for the same combination of os/cuda/mofed. it just creates confusion about which container should be used. can we add these links and valgrind package to the existing centos7-cuda11-mofed5?

Reply:
ok, no problems